### PR TITLE
Add logging when filestore operations fail

### PR DESCRIPTION
**Pull request by kdmccormick**
#219 is bugged. If you try to download or upload enrollments on Stage, the job fails. Looking at Splunk, an AccessDenied is happening when trying to save to S3. I'm thinking that the bucket name might be misconfigured.

This PR adds logging to help diagnose it.

**Bonus:** Fixes our only RemoveInDjango20Warning

### CODE REVIEW

File: registrar/apps/core/models.py [hunk 1]
```diff
@@ -122,6 +122,8 @@ class OrganizationGroup(Group):
 
     .. no_pii::
     """
+    objects = models.Manager()
```
Comment (michaelroytman) [inline registrar/apps/core/models.py:125]: Is this related to the logging?

**Review by michaelroytman — APPROVED** [review]

File: registrar/apps/core/models.py (hunk 1 above)
Comment (kdmccormick) [reply registrar/apps/core/models.py:125]: Nah, it's to silence a RemovedInDjango20Warning. It's in its own commit.
